### ISSUE TITLE
Hanging on green spinning circle when I deploy a bot

### ISSUE THREAD

**Issue by markstur** [IBM/chatbot-deployer#37]
The Watson Assistant auth changes might have broken the deploy.

**Comment by stevemar** [comment]
The usually automatic deployment didn't kick off for me 

<img width="761" alt="screen shot 2018-07-07 at 11 27 02 pm" src="https://user-images.githubusercontent.com/4118756/42416477-51fb4b36-823d-11e8-96e9-a1fc46795f9f.png">

**Comment by stevemar** [comment]
Once I hid the play button it looks like it worked

```
Preparing to start the job...
Logging into Bluemix as user: "stevemar@ca.ibm.com" 
Target: https://api.ng.bluemix.net
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   411  100   411    0     0    917      0 --:--:-- --:--:-- --:--:--   919

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 27305  100 27305    0     0  27547      0 --:--:-- --:--:-- --:--:-- 77351
Creating new conversation service ... 
Creating service instance Bot Asset Exchange Workspaces in org stevemar@ca.ibm.com / space dev as stevemar@ca.ibm.com...
OK
Creating service key Bot Asset Exchange Workspaces for service instance Bot Asset Exchange Workspaces as stevemar@ca.ibm.com...
OK
=============================================================
Created new conversation service: SERVICE_NAME: Bot Asset Exchange Workspaces
=============================================================
/home/pipeline/.npm-global/bin/json -> /home/pipeline/.npm-global/lib/node_modules/json/lib/json.js
json@9.0.6 /home/pipeline/.npm-global/lib/node_modules/json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 27305    0     0  100 27305      0   111k --:--:-- --:--:-- --:--:--  112k
100 27674  100   369  100 27305    805  59622 --:--:-- --:--:-- --:--:-- 59617
{
  "name": "Travel Booking",
  "created": "2018-07-08T03:27:58.056Z",
  "updated": "2018-07-08T03:27:58.056Z",
  "language": "en",
  "metadata": {
    "api_version": {
      "major_version": "v1",
      "minor_version": "2017-05-26"
    }
  },
  "description": "Bot for booking flight,cab and knowing status of flight!",
  "workspace_id": "0836d8ca-93d0-44d5-bf52-6d603f59ecf3",
  "system_settings": null,
  "learning_opt_out": false
}
==================================================
Created new workspace: WORKSPACE_ID: 0836d8ca-93d0-44d5-bf52-6d603f59ecf3
==================================================
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   389  100   389    0     0   1305      0 --:--:-- --:--:-- --:--:--  1309
{
  "name": "Travel Booking",
  "created": "2018-07-08T03:27:58.056Z",
  "updated": "2018-07-08T03:27:58.056Z",
  "language": "en",
  "metadata": {
    "api_version": {
      "major_version": "v1",
      "minor_version": "2017-05-26"
    }
  },
  "description": "Bot for booking flight,cab and knowing status of flight!",
  "workspace_id": "0836d8ca-93d0-44d5-bf52-6d603f59ecf3",
  "system_settings": null,
  "learning_opt_out": false,
  "status": "Training"
}

Finished: SUCCESS
```

**Comment by stevemar** [comment]
Another quirk... For some reason searching for "Bot Asset Exchange Workspaces" doesn't result in the service showing up. Despite the fact that it worked before.

<img width="1046" alt="screen shot 2018-07-07 at 11 30 05 pm" src="https://user-images.githubusercontent.com/4118756/42416493-ce9fd242-823d-11e8-9f02-7bf2ad95e9a5.png">
<img width="1388" alt="screen shot 2018-07-07 at 11 29 48 pm" src="https://user-images.githubusercontent.com/4118756/42416494-ceae7400-823d-11e8-8596-5ab6f5a09741.png">

**Comment by stevemar** [comment]
With all that said, it worked for me if I hit play and knew what I was doing. See screenshot.

<img width="429" alt="screen shot 2018-07-07 at 11 32 55 pm" src="https://user-images.githubusercontent.com/4118756/42416505-222c543a-823e-11e8-9f10-39c1665bb713.png">

We should open (or re-open) a bug against the toolchain team to make the script automatically run. 

Here's my original bug: https://github.ibm.com/org-ids/roadmap/issues/3427 and their fix: https://github.ibm.com/org-ids/pipeline-service/pull/213

I have no idea who to bug about the search being broken (this affects any search with a space in the service name).

**Comment by skaegi** [comment]
Hrm.. not sure what's going wrong. If I had to guess I'd say a permissions problem with your git repo's webhook. The easiest way to fix this is probably to do a screenshare with me. Ping me in Slack (@skaegi)

**Comment by stevemar** [comment]
This was mostly fixed by #39 but we should update the README to explain the non-obvious parts.
